src/shared/components/palette/address/write-address-field.component.ts

```diff
@@ -78,7 +80,18 @@ export class WriteAddressFieldComponent extends AbstractFieldWriteComponent impl
     this.setFormValue();
   }
 
+  isComplexWithHiddenFields() {
+    if (this.caseField.field_type
```

Comment:
this.caseField.field_type is not needed as field_type is mandatory

Reply:
👍